app/main.py

```diff
@@ -1,3 +1,51 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __add__(self, other: Distance) -> Distance:
```

Comment:
The type annotation for the 'other' parameter in the __add__ method should include both 'Distance' and 'Union[int, float]' to reflect that it can accept either a Distance object or a numeric type.